build.sbt

```diff
@@ -89,7 +93,7 @@ lazy val root = (project in file("."))
         scalaVersion := "2.13.2",
         crossScalaVersions += "2.12.10",
         resolvers ++= Seq(
-          Resolver.bintrayRepo("ovotech", "maven")
+          "Artifactory" at "https://kaluza.jfrog.io/artifactory/maven",
```

Comment:
Not  a big issue but I would personally move this url to a val so that is only in one place